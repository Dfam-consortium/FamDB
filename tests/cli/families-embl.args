families
--format
embl
-d
2
