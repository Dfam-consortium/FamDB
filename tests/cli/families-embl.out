CC   <copyright header>
ID   DR000000001; SV 1; linear; DNA; STD; UNC; 8 BP.
NM   Test family DR000000001
XX
AC   DR000000001;
XX
XX
KW   /.
XX
OS   Other Species
OC   root; Order; Other Genus.
XX
CC
CC   RepeatMasker Annotations:
CC        Type: 
CC        SubType: 
CC        Species: Other_Species
CC        SearchStages: 
CC        BufferStages: 
XX
SQ   Sequence 8 BP; 2 A; 2 C; 2 G; 2 T; 0 other;
     gcatatcg                                                           8
//
ID   DR_Repeat1; SV 1; linear; DNA; STD; UNC; 7 BP.
NM   Test family DR_Repeat1
XX
AC   DR_Repeat1;
XX
XX
KW   /.
XX
OS   Species
OC   root; Order; Genus.
XX
CC
CC   RepeatMasker Annotations:
CC        Type: 
CC        SubType: 
CC        Species: Species
CC        SearchStages: 
CC        BufferStages: 
XX
SQ   Sequence 7 BP; 2 A; 2 C; 1 G; 2 T; 0 other;
     cgactat                                                            7
//
ID   TEST0004; SV 1; linear; DNA; STD; UNC; 8 BP.
NM   Test family TEST0004
XX
AC   TEST0004;
XX
XX
KW   SINE/.
XX
OS   Genus
OC   root; Order.
XX
CC
CC   RepeatMasker Annotations:
CC        Type: SINE
CC        SubType: 
CC        Species: Genus
CC        SearchStages: 35
CC        BufferStages: 10[1-2],10[5-8],20
XX
SQ   Sequence 8 BP; 0 A; 4 C; 0 G; 4 T; 0 other;
     cccctttt                                                           8
//
